piqueserver/extensions.py

```diff
@@ -63,25 +62,24 @@ def load_scripts(script_names, script_dir, script_type):
     '''
     script_objects = []
 
-    for script in script_names[:]:
+    finder = importlib.machinery.PathFinder()
+    for script in script_names:
+        spec_scripts = finder.find_spec(script, [script_dir])
+        spec_global = importlib.util.find_spec(script)
+        spec = spec_scripts or spec_global
+        if not spec:
+            log.error(
+                "{} '{}' not found in either {} directory or global scope".format(
+                    script_type, script, script_dir))
+            continue
         try:
-            # this finds and loads scripts directly from the script dir
-            # no need for messing with sys.path
-            f, filename, desc = imp.find_module(script, [script_dir])
-            module = imp.load_module('piqueserver_{}_namespace_'.format(
```

Comment:
the removal of the namespace modification will break script names that shadow global module names again :/